### ISSUE TITLE
CI: builds failed due to k8s 1.12 manifests not including fixed-identity for kube-dns

### ISSUE THREAD

**Issue by ianvernon** [cilium/cilium#5793]
This is a tracking ticket for recent build failures in the CI.

etcd-operator support was merged into the CI. Right after, the K8s versions were bumped in the CI as well. K8s 1.12 manifests did not include the fixed-identity for kube-dns, resulting in Cilium not being able to bootstrap because it could not access kube-dns.

**Comment by ianvernon** [comment]
#5790 is out to fix this. #5789 is also running in parallel to revert the K8s version bumps in the CI in case #5790 does not work, in order to unblock the CI as soon as possible.

**Comment by ianvernon** [comment]
Fixed-by #5790
